cmd/test/main_test.go

```diff
@@ -240,6 +252,49 @@ func test_testCommand_version_flags_validation(t *testing.T, ctx *TestCommandCon
 	assert.NoError(t, err)
 }
 
+func test_testCommand_empty_dir(t *testing.T, readerMock *ReaderMock, ctx *TestCommandContext) {
+	emptyDir := t.TempDir()
+	emptyDirPaths := filepath.Join(emptyDir, "*.yaml")
+
+	readerMock.On("FilterFiles", []string{emptyDirPaths}).Return([]string{}, nil)
+	err := executeTestCommand(ctx, []string{emptyDirPaths})
+
+	assert.EqualError(t, err, "No files detected")
+	readerMock.AssertCalled(t, "FilterFiles", []string{emptyDirPaths})
+}
+
+func test_testCommand_no_valid_k8s_resources(t *testing.T, readerMock *ReaderMock, evaluator *mockEvaluator, k8sValidator *K8sValidatorMock, filesConfigurations []*extractor.FileConfigurations, printerMock *PrinterMock, ctx *TestCommandContext) {
+	testPath := pathFromRoot("internal/fixtures/nonKube/*.(yaml|json)")
+	readerMock.On("FilterFiles", []string{testPath}).Return([]string{testPath}, nil)
```

Comment:
I added the `readerMock` line here because I wanted to test if the `FilterFiles` function actually get the `testPath` parameter while executing the `datree test "./internal/fixtures/nonKube/*.(yaml|json)"` command. Else the mocked `FilterFiles` function always trigger a `AssertCalled` event with `8/*` parameter.